pocketid-provider/README.md

```
@@ -0,0 +1,12 @@
+# Pocket ID Provider (by Ebnater)
+
+This plugin allows you to use PocketID as OAuth Provider.
+
+## Features
+
+- Register PocketID as OAuth Provider
+- Integrates with the already known OAuth Tab in Pelican Panel Settings
+
+## Credits to devilr33f
+
+I used his `PocketIDSchema.php` file from https://github.com/devilr33f/
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)